base/pom.xml

```diff
@@ -50,6 +50,9 @@ ${project.groupId}.annotation.*;version=${project.version}
       <plugin>
         <groupId>org.moditect</groupId>
         <artifactId>moditect-maven-plugin</artifactId>
+        <configuration>
```

Comment:
I moved this to parent pom, so should not be needed here (at least seemed to work when I tried)

Reply:
wasn't placing in the META-INF/vesrions folder :)

Reply:
hmmh. Seemed to, for me, but I can double-check after merging.